server/config/game/specialists.json

```diff
@@ -420,34 +422,34 @@
             "id": 20,
             "key": "metroid",
             "name": "Experimental Lifeform",
-            "description": "When attacking a star, adds +6 Weapons to your side in combat. Does not participate in carrier-to-carrier combat. Dies after 20 ticks. Gains no rewards from invading.",
+            "description": "When attacking a star, adds +10 Weapons to your side in combat. When attacking in hyperspace, -2 Weapons to your side in combat. Dies after 30 ticks. Gains no rewards from invading.",
```

Comment:
I'm gonna input that +10 weapons for that small of tokens is too much;
- Does not fit with all other specialists
- Completely waters down to he weapons tech, we want specialists to give slights edges on your current weapons tech, not replace it completely

Reply:
I understand your concerns about the +10 weapons bonus for a low token cost on this specialist. Here's the rationale behind it: I designed this specialist to offer a unique strategic choice, specializing in tech over weapons. It's an unstable, powerful lifeform with a random lifespan (5 to 35 ticks) and a focus on invading other star systems. This is just running further along with the original design from Solaris.

To avoid overshadowing other aspects of the game, we can adjust the weapons bonus and increase the token cost. I think this will maintain balance and make this carrier actually feel unique. I like the risk-reward nature of this specialist, where players may use it strategically, even if it eventually dies and costs them tokens.

One balance tweak was giving it a -2 weapons penalty while traveling through hyperspace, making it more vulnerable to interception. 

Previously it was just a slightly stronger specialist that dies pretty quickly and something I saw no one use, nor would I consider using outside of testing.

Let me know your thoughts and any specific changes you'd suggest.